public/video-ui/src/actions/UploadActions/s3Upload.js

```diff
@@ -26,9 +26,15 @@ function uploadComplete() {
 
 export function startUpload(id, file, completeFn) {
   return dispatch => {
+    // Start prompting the user about reloading the page
+    window.onbeforeunload = () => { return false; };
```

Comment:
Browsers remove support for that. See "Custom text support removed" here: https://developer.mozilla.org/en-US/docs/Web/API/WindowEventHandlers/onbeforeunload#Browser_compatibility